packages/requisitions/src/ResponseRequisition/DetailView/AdditionalInfoSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
```

Comment:
just lifted the components out.. making it consistent with the other side panels